docker/php/Dockerfile

```
@@ -36,7 +36,7 @@ RUN mv composer.phar /usr/local/bin/composer
 
 RUN gem install sass -v 3.4.23
 RUN gem install compass
-RUN npm install -g grunt-cli@1.3.2
+RUN npm install -g grunt-cli
```

```suggestion
RUN npm install -g grunt-cli@4.0.0
```
Shouldn't we lock, just in case?

---

The v4.0.0 version is the one of grunt-contrib-uglify. We can lock the version of grunt-cli to v1.4.1, since the 1.4.2 bumps the minimum version of node to v10.

---

Also, maybe we can move this dependency from the Dockerfile to the package.json?

---

we need it globally as it is used as installed program.
I'd require here the exact same version specified in package.json